docs/developers_guide/machines/index.rst

```diff
@@ -16,53 +16,57 @@ cases are configured in a way that is appropriate for your machine.
+    source test_compass_1.0.0_anvil_intel18_mvapich.sh
 
 After loading this environment, you can set up test cases or test suites, and
 a link ``load_compass_env.sh`` will be included in each suite or test case
-work directory.  This is a link to a specific activation script for that
-machine and compiler (so not a link to ``load/load_compass_env.sh``, it just
-happens to be given the same name).  You can can source this file on a
-compute node (e.g. in a job script) to get the right compass conda environment,
-compilers, MPI libraries and environment variables for running MPAS.
+work directory.  This is a link to the activation script that you sourced when
+you were setting things up.  You can can source this file on a compute node
+(e.g. in a job script) to get the right compass conda environment, compilers,
+MPI libraries and environment variables for running``compass`` tests and
+the MPAS model.
+
++--------------+------------+-----------+-------------------+
+| Machine      | Compiler   | MPI lib.  |  MPAS make target |
++==============+============+===========+===================+
+| anvil        | intel18    | mvapich   | ifort             |
+|              +------------+-----------+-------------------+
+|              | gnu        | mvapich   | gfortran          |
++--------------+------------+-----------+-------------------+
+| badger       | intel      | impi      | intel-mpi         |
+|              +------------+-----------+-------------------+
+|              | gnu        | mvapich   | gfortran          |
++--------------+------------+-----------+-------------------+
+| chrysalis    | intel      | openmpi   | ifort             |
+|              |            +-----------+-------------------+
+|              |            | impi      | intel-mpi         |
+|              +------------+-----------+-------------------+
+|              | gnu        | openmpi   | gfortran          |
++--------------+------------+-----------+-------------------+
+| compy        | intel      | impi      | intel-mpi         |
+| (fix coming  +------------+-----------+-------------------+
+| soon)        | pgi        | mvapich2  | pgi               |
++--------------+------------+-----------+-------------------+
+| cori-haswell | intel      | mpt       | intel-nersc       |
+|              +------------+-----------+-------------------+
+|              | gnu        | mpt       | gnu-nersc         |
++--------------+------------+-----------+-------------------+
+| cori-knl     | intel      | impi      | intel-mpi         |
+|              +------------+-----------+-------------------+
+|              | gnu        | mpt       | gnu-nersc         |
++--------------+------------+-----------+-------------------+
+| grizzly      | intel      | impi      | intel-mpi         |
+|              +------------+-----------+-------------------+
+|              | gnu        | mvapich   | gfortran          |
++--------------+------------+-----------+-------------------+
```

Comment:
@vanroekel, @matthewhoffman and @mark-petersen, you all had similar, excellent suggestions so here's a table of the supported compilers, MPI libraries and make targets

Reply:
@xylar this table is a great addition, thanks for adding that.